antarest/study/storage/variantstudy/model/command/create_link.py

```diff
@@ -185,22 +186,66 @@ def _apply(self, study_data: FileStudy) -> CommandOutput:
         study_data.tree.save(
             link_property, ["input", "links", area_from, "properties", area_to]
         )
-        if self.series:
-            assert isinstance(self.series, str)
+        if self.series is None:
+            self.series = (
+                self.command_context.generator_matrix_constants.get_link(
+                    version=version
+                )
+            )
+
+        assert type(self.series) is str
+        if version < 820:
             study_data.tree.save(
                 self.series, ["input", "links", area_from, area_to]
             )
+        else:
+            study_data.tree.save(
+                self.series,
+                ["input", "links", area_from, f"{area_to}_parameters"],
+            )
+            if self.direct:
```

Comment:
there should be a default series for direct and indirect I guess (to test with AntaresSimulator)